crypto/ocsp/ocsp_verify.c

```diff
@@ -0,0 +1,359 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <string.h>
+#include "internal.h"
+
+// supporting internal static functions for OCSP_basic_verify
+
+#define IS_OCSP_FLAG_SET(flags, query) (flags & query)
+// find signer in cert stack or |OCSP_BASICRESP|'s cert stack
+static int ocsp_find_signer(X509 **psigner, OCSP_BASICRESP *bs, STACK_OF(X509) *certs, unsigned long flags);
```

Comment:
Typically, we just define static functions before they are used without the need for declarations. Why change this pattern?